category_encoders/quantile_encoder.py

```diff
@@ -0,0 +1,291 @@
+"""Target Encoder"""
+__author__ = "david26694", "cmougan"
+
+import numpy as np
+import pandas as pd
+from sklearn.base import BaseEstimator
+from category_encoders.ordinal import OrdinalEncoder
+from category_encoders.m_estimate import MEstimateEncoder
+import category_encoders.utils as util
+from sklearn.utils.random import check_random_state
+class QuantileEncoder(BaseEstimator, util.TransformerWithTargetMixin):
```

Comment:
Please add 2 blank lines (PEP-8) before the class definition. Also consider just formatting using black (which you actually did right? maybe that got through somehow- https://github.com/psf/black).

Reply:
Would this be the formatting?
 `python -m black category_encoders/quantile_encoder.py`